packages/docusaurus-plugin-vercel-analytics/src/plugin-vercel-analytics.d.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+declare module '@docusaurus/plugin-vercel-analytics' {
```

Comment:
Move plugin options types to `options.ts`. 

Instead of declaring types, you can let typescript generate the typedef for you (see what other gtag plugins are doing)

Use this instead at the end of `index.ts`

`export type {PluginOptions, Options};`

Those would end up in `index.d.ts` 
(which is the generated def for `@docusaurus/plugin-vercel-analytics`, cf `"types": "index.d.ts"` in package.json)

Reply:
shouldn't this file be deleted now?